x/ccv/provider/types/proposal.go

```diff
@@ -198,6 +203,41 @@ func (sccp *ConsumerRemovalProposal) ValidateBasic() error {
 	return nil
 }
 
+// NewEquivocationProposal creates a new equivocation proposal.
+// [DEPRECATED]: do not use since that equivocations can be submitted
```

Comment:
```suggestion
// [DEPRECATED]: do not use because equivocations can be submitted
```
I don't get the comment, is the suggestion what it's supposed to say?